README.md

```diff
@@ -15,6 +15,31 @@ The Dockerfile will create a postgres database (pg-data) and redis (redis-data)
 
 To enter the container in a bash environment, run `bin/docker bash`. To enter the postgres container using psql, run `bin/docker pg`. You can run rails commands with `bin/docker rails [command to run]`. Run `bin/docker` alone to see all the available options.
 
+### Replicating the staging database on a Dockerized local server:
+
+To get a dump of the most recent version of the staging database, run:
+
+``` 
+heroku pg:backups:capture -a decidim-seattle-staging
```

Comment:
I think it's fine to have the app name here, but you could also write the command more generically if you like: `heroku pg:backups:capture -a app-name` and then specify that the app name in our case is `decidim-seattle-staging`.